include/accelerator/rectifier.hpp

```diff
@@ -30,20 +28,32 @@ enum class MappingImpl {
     OpenCV
 };
 
+class RectifierBase {
+public:
+    virtual ~RectifierBase() {}
+    virtual Image::UniquePtr rectify(const Image &msg) = 0;
+    CameraInfo GetCameraInfoRect(void) {
+        // return copy of the current camera info rect
+        return CameraInfo(camera_info_rect_);
+    }
+
+protected:
+    CameraInfo camera_info_rect_{};
```

Comment:
Would it be safer to use `std::optional` for `camera_info_rect_`?
For the case when `camera_info_rect_` does not exist or has not been set yet, to avoid getting a nasty surprise if `GetCameraInfoRect` is called too early.